src/v2/styles/Avatar/Avatar-theme.scss

```diff
@@ -41,6 +41,7 @@
   .str-chat__avatar-fallback {
     @include utils.component-layer-overrides('avatar');
     text-transform: uppercase;
+    font-size: 20px;
```

Comment:
I think this should be in the layout file, and computed from the `--str-chat__avatar-size` variable